test/functional/rpc_help.py

```diff
@@ -32,7 +32,7 @@ def process_mapping(fname):
                 if line.startswith('};'):
                     in_rpcs = False
                 elif '{' in line and '"' in line:
-                    m = re.search(r'{ *("[^"]*"), *([0-9]+) *, *("[^"]*") *},', line)
+                    m = re.search(r'{ *("[^"]*"), *([0-9]+) *, *("[^"]*") *(, \/\*also_string=\*\/true *)?},', line)
```

Comment:
Might be a good idea to actually test this is correct?

Reply:
This is here because the test fails without it.

Reply:
Following your suggestion @ryanofsky... If a new type is added (`NUM_OR_STR` or even `ARR_OR_STR` from #32468), is `also_string` still needed?

Reply:
re: https://github.com/bitcoin/bitcoin/pull/33230#discussion_r2376072486

> Following your suggestion @ryanofsky... If a new type is added (`NUM_OR_STR` or even `ARR_OR_STR` from #32468), is `also_string` still needed?

Yes my suggestion above is just a way of improving this python test, which makes sure the client-side type table in `src/rpc/client.cpp` is consistent with type information in the server RPC code. The other file `src/rpc/util.cpp` is a server-side file while provides information about the server parameter types to the python test. So an `also_string` column (or equivalent) is still needed in `src/rpc/client.cpp` even if the test is improved.